.gitignore

```diff
@@ -23,3 +23,6 @@ gen
 *.bat
 config.toml
 exp.rs
+
+# Mac specific
+.DS_Store

```

Comment:
FYI you can add this to your `~/.gitconfig` to avoid needing to add this to every project's .gitignore
```
[core]
	excludesfile = /home/<USER>/.gitignore
```

Reply:
I personally don't really mind this, as it's so common. If it achieves that not every single macOS contributor has to modify their global `.gitconfig`, then that's a win in my eyes.